bigtable/tableadmin/tableadmin_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import random
+
+from tableadmin import delete_table
+from tableadmin import run_table_operations
+
+PROJECT = os.environ['GCLOUD_PROJECT']
```

Comment:
Please confirm that the tests pass before this PR is merged. @andrewsg should be able to point you to any environment variables that might need to be set.